From 4fa680e63ff1b4f35404d108408521adf30a6197 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Sun, 19 Aug 2018 15:30:27 +0200
Subject: [PATCH 034/293] xlat: add IPV4_DEVCONF_BC_FORWARDING-1 to
 inet_devconf_indices.in

* xlat/inet_devconf_indices.in (IPV4_DEVCONF_BC_FORWARDING-1): New
constant, introduced by Linux commit v4.19-rc1~140^2~208^2~1.
---
 xlat/inet_devconf_indices.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xlat/inet_devconf_indices.in b/xlat/inet_devconf_indices.in
index eabcfc5..dd26237 100644
--- a/xlat/inet_devconf_indices.in
+++ b/xlat/inet_devconf_indices.in
@@ -31,3 +31,4 @@ IPV4_DEVCONF_IGMPV3_UNSOLICITED_REPORT_INTERVAL-1	28
 IPV4_DEVCONF_IGNORE_ROUTES_WITH_LINKDOWN-1		29
 IPV4_DEVCONF_DROP_UNICAST_IN_L2_MULTICAST-1		30
 IPV4_DEVCONF_DROP_GRATUITOUS_ARP-1			31
+IPV4_DEVCONF_BC_FORWARDING-1				32
-- 
1.7.12.4

